### PR TITLE
[Raptor] ignore the max_extra_second_pass parameter and set it to zero

### PR DESCRIPTION
**Pull request by pbench**
To be merged with https://github.com/CanalTP/artemis_references/pull/226

### CODE REVIEW

**Review by xlqian — APPROVED** [review]

File: source/kraken/worker.cpp [hunk 1]
```diff
@@ -751,14 +751,14 @@ void Worker::journeys(const pbnavitia::JourneysRequest& request, pbnavitia::API
                                            "no origin point nor destination point given");
             return;
         }
-
+        uint32_t max_extra_second_pass = 0;  // request.max_extra_second_pass()
```
Comment (kadhikari) [inline source/kraken/worker.cpp:754]: We don't use any more the parameter  max_extra_second_pass ?

Comment (pbench) [reply source/kraken/worker.cpp:754]: It is to deploy the raptor PR on all plaforms (dev, internal, etc.) without having to update the navitia-deployment-conf.
With the PR, the parameter should be zero, without the PR we want to keep using the parameters in navitia-deployment-conf.

**Review by benoit-bst — APPROVED** [review]

**Review by kadhikari — APPROVED** [review]

**Review by nonifier — APPROVED** [review]
Can we track somewhere to revert this once deployed ? 
We are so good at forgetting things like this :smile:

**Comment by pbench** [comment]
> Can we track somewhere to revert this once deployed ?
> We are so good at forgetting things like this smile

I'll make a ticket :)

**Pull request merged**